src/style.css

```diff
@@ -1,3 +1,96 @@
-body {
+@font-face {
+  font-family: Roboto;
+  src: url('/fonts/Roboto-Medium.ttf') format('truetype');
+  font-weight: 500;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family:
+    Roboto,
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    Arial,
+    sans-serif;
+}
+
+header {
```

Comment:
fix it everywhere
<img width="852" alt="image" src="https://github.com/user-attachments/assets/8a8cf4e0-e504-421c-bc8f-74f68b62a808">